### PR TITLE
enable service bus and use one blob storage

### PR DESCRIPTION
**Pull request by mdayican**
### Change description ###

enable service bus and use one blob storage

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/demo/common/bsp/bsp-sample-app.yaml [hunk 1]
```diff
@@ -60,6 +60,16 @@ spec:
     bulk-scan-processor:
       java:
         image: hmctspublic.azurecr.io/bulk-scan/processor:prod-732e2ec3
+        secrets:
```
Comment (lgonczar) [inline k8s/demo/common/bsp/bsp-sample-app.yaml]: Why do we want these secrets here and not in bsp-demo?

Comment (mdayican) [reply k8s/demo/common/bsp/bsp-sample-app.yaml]: do you mean why no in product chart?

Comment (doncem) [reply k8s/demo/common/bsp/bsp-sample-app.yaml]: yes. in order to create a sample app product chart one needs to execute several steps and allocate resources we don't actually need - it's a sample app. we should not care about it. just a proof e2e actually works giving the configuration

Comment (mdayican) [reply k8s/demo/common/bsp/bsp-sample-app.yaml]: still we need to manage dependencies and configure them. 
or we can use bulk-scan product chart and add sample app.

Comment (lgonczar) [reply k8s/demo/common/bsp/bsp-sample-app.yaml]: 👍

**Comment by mdayican** [comment]
I merging this, we will discuss tomorrow  .

**Pull request merged**